css/style.css

```diff
@@ -112,3 +112,283 @@ a {
+  margin: 0 0 30px 0;
+}
+
+/** ABOUT SECTION **/
+
+.about-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 50px 0 0 0;
+}
+
+.about-title {
+  text-align: center;
+  margin-top: 40px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 10px;
+  color: #172b4d;
+}
+
+.about-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(../img/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.about-content p {
+  font-size: 25px;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  padding: 0 25px 0 25px;
+}
+
+.about-content button {
+  background-color: #36b37f;
+  padding: 12px 18px;
+  border-radius: 4px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  margin: 20px 0 30px 0;
+}
+
+.about-section ul {
+  padding: 0;
+}
+
+.about-card-cont {
+  display: grid;
+  grid-template-columns: 1fr;
+  column-gap: 24px;
+}
+
+.about-card li {
+  list-style: none;
+}
+
+.about-card {
+  width: 327px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 30px 0 20px 0;
+  border: 1px solid #c1c7d0;
+}
+
+.about-card-logo-1 {
+  height: 80px;
+  width: 80px;
+  background-color: #fff;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 40px 0 54px 0;
+}
+
+.about-card-logo-2 {
+  height: auto;
+  width: auto;
+  background-color: #fff;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 40px 0 54px 0;
+}
+
+.about-card-logo-2 img {
+  padding: 16px;
+}
+
+.about-card-logo-3 {
+  height: auto;
+  width: auto;
+  background-color: #fff;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 40px 0 54px 0;
+}
+
+.about-card-logo-3 img {
+  padding: 16px;
+}
+
+.about-card h2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+  margin: 20px 0 30px 0;
+}
+
+.about-card-pr-lang {
+  padding: 0;
+  gap: 8px 8px;
+  margin: 0 0 30px 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.about-card-pr-lang li {
+  list-style: none;
+  padding: 12px;
+  border-radius: 8px;
+  background-color: #fff;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #36b37e;
+}
+
+/** HOVER EFFECTS **/
+
+.work-card-info button:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.work-card-info button:active {
+  background-color: #008552;
+}
+
+.about-content button:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.about-content button:active {
+  background-color: #008552;
+}
```

Comment:
**[OPTIONAL]:** You could summarize these code and make it cleaner :sparkles:  by chaining selectors like so: 

```
work-card-info button:hover,
.about-content button:hover
 {
  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
}

.work-card-info button:active,
.about-content button:active {
  background-color: #008552;
}
```

**Note:** This is only a suggestion and It's the concept of a principle you will learn in further lessons called DRY. You can read a little about it [here :link: ](https://www.digitalocean.com/community/tutorials/what-is-dry-development?refcode=49d9dff6b635&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=CopyPaste)

Reply:
Thank you for the suggestion. This has been updated accordingly.